python/sglang/srt/server_args.py

```diff
@@ -401,6 +402,11 @@ def __post_init__(self):
             print_deprecated_warning(
                 "NOTE: --enable-triton-kernel-moe is deprecated. Please set `--moe-runner-backend` to 'triton_kernel' instead."
             )
+        if self.enable_flashinfer_cutedsl_moe:
+            self.moe_runner_backend = "flashinfer_cutedsl"
+            print_deprecated_warning(
+                "NOTE: --enable-flashinfer-cutedsl-moe is deprecated. Please set `--moe-runner-backend` to 'flashinfer_cutedsl' instead."
+            )
```

Comment:
we don't need this deprecation warning, right? since the --enable-flashinfer-cutedsl-moe is never a thing before.

Reply:
for consistency maybe we need